sdk/src/hex/bytes.ts

```diff
@@ -0,0 +1,43 @@
+import {InvalidHexStringError, OddLengthHexStringError} from './errors';
+import {ensure0x, isHexString, strip0x} from './strings';
+
+/**
+ * Encodes a buffer into a hex string with the "0x" prefix
+ *
+ * @export
+ * @param {string} hexString
+ * @return {Uint8Array}
+ */
+export function hexToBytes(hexString: string): Uint8Array {
+  if (!hexString) return new Uint8Array();
```

Comment:
If the possibility of hexString is to be a non-falsy string value then the param should reflect that. Else explicitly check for  `=== ''`